packages/cli/package.json

```diff
@@ -1,8 +1,8 @@
 {
   "name": "@spaship/cli",
   "description": "A command line interface for SPAship!",
-  "version": "1.3.0",
-  "author": "mclayton@redhat.com",
+  "version": "1.3.1",
+  "author": "Soumyadip Chowdhury",
```

Comment:
@soumyadip007 you can add your name in the contributors list

Reply:
Resolved